### ISSUE TITLE
webtoon Source directory is empty

### ISSUE THREAD

**Issue by xMonTersx** [ciromattia/kcc#452]
I have used KCC for long time. But recently I got an error call "Source directory is empty". But there are number of images file in the selected directory.

I dont know why, could anyone help me with this
I attatched some of my screenshot
![image](https://user-images.githubusercontent.com/10475907/209905224-518f0a44-a89a-460f-9cc6-b132723373dd.png)
![image](https://user-images.githubusercontent.com/10475907/209905310-739bd53f-091e-4b9b-8f00-819f6db9225c.png)

**Comment by paolovinoya** [comment]
Since you were only converting a single directory, have you tried to compress to cbz and try again?

**Comment by xMonTersx** [comment]
> Since you were only converting a single directory, have you tried to compress to cbz and try again?

Yes, I had already done that and nothing help. I even try to convert image to png or something else. But it did not work

**Comment by xMonTersx** [comment]
@paolovinoya 
You mean this
![image](https://user-images.githubusercontent.com/10475907/211230419-21817a31-01e3-4e70-b259-a86ac88327f8.png)
As you can see, there are images in this folder @@

**Comment by paolovinoya** [comment]
Is it necessary to keep the apostrophe on your directory? It could lead to string termination issues.

![image](https://user-images.githubusercontent.com/26787363/211239410-78d66b05-c198-41bf-b298-da2f83be9065.png)

Is your computer using Windows permissions? Try checking if the directory is set with permissions that an application cannot enter or browse through.
`icacls "D:\Manhwa\girls's secret\Chuong 24 End\"`

**Comment by xMonTersx** [comment]
@paolovinoya 
I rename all with the no space and no special characters, and nothing work.
![image](https://user-images.githubusercontent.com/10475907/211240071-f4b6a849-0d58-4b9e-8d9a-af3195bf85a4.png)
![image](https://user-images.githubusercontent.com/10475907/211240083-37b0d6a6-c2ce-40fd-91bc-b6037892bfe4.png)
I found that this error only happen in webtoon mode, everything will go well in manga mode
![image](https://user-images.githubusercontent.com/10475907/211240677-c19c9e33-bd61-4290-a17b-46a96eb65f47.png)

**Comment by paolovinoya** [comment]
The errors are probably found on the console. Try a command line to see any errors thrown out into the console.

`kcc-c2e -p K34 -w -s --forcecolor "D:\dir\location\of\manga"`
These settings provided are the exact switches that you chose on the GUI.
-w - Webtoon
-s Stretch and Upscale
--forcecolor - Color mode
-p K34 and using profile Kindle Paperwhite 3/4 

If no errors, try also another commandline tool

`kcc-c2p -y 1448 -m -d`
This command will throw out debug files on which image it will plan to crop/cut to compile into the epub/mobi file.

**Comment by axu2** [comment]
Are you able to run the latest master from source? It has improved error handling and might make a difference.
Or you can fork the repo and run the GitHub Action to generate an exe.

**Comment by axu2** [comment]
@xMonTersx I think I fixed this? https://github.com/axu2/kcc/releases/tag/v9.1.0-webtoon-empty

Otherwise, can you provide a sample file with the issue?

**Comment by axu2** [comment]
Ok another user confirmed it's related to one drive on windows 11. Temporary fix by disabling it I'll see if proper fix is possible 

tell me the output of 

echo %TMPDIR%
echo %TEMP%
echo %TMP%


I'm curious if onedrive shows up there.

Edit: I think I've found the real problem...

**Comment by axu2** [comment]
@xMonTersx 

It's finally fixed in above pr, test builds linked.

**Comment by xMonTersx** [comment]
> [@xMonTersx](https://github.com/xMonTersx)
> 
> It's finally fixed in above pr, test builds linked.

Thanks bro
